src/sdk/pynni/nni/compression/torch/builtin_quantizers.py

```diff
@@ -21,6 +21,19 @@ def quantize_weight(self, weight, config, op_name, **kwargs):
         orig_type = weight.type()  # TODO: user layer
         return weight.div(scale).type(torch.int8).type(orig_type).mul(scale)
 
+# class EMA_RangeChecker:
+#     def __init__(self, alpha):
+#         self.alpha = alpha
+#         self.args = None
+
+#     def update(self, args):
+#         if self.args is None:
+#             if not isinstance(args, list):
+#                 raise TypeError("expect type list of parameter args")
+#             self.args = args
+#         else:
+#             self.args = [self.alpha * args[idx] + (1 - self.alpha) * self.args[idx] for idx in range(len(self.args))]
+#         return self.args
```

Comment:
since it is commented, why not directly remove? or it will be used in future?

Reply:
Yes, it will be used in the future.